Application/Application.csproj

```diff
@@ -16,6 +16,7 @@
 
     <ItemGroup>
         <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="10.0.1" />
+        <PackageReference Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="2.2.0" />
```

Comment:
Is this added for the "FromQuery" attribute?  Just wondering if there's a way we can do it without adding aspnet dll's in to the application layer?

Reply:
done!